src/loot_filter.py

```diff
@@ -37,7 +37,7 @@ def check_items(inv: InventoryBase, force_refresh: ItemRefreshType):
     start_time = None
     img = None
     for item in occupied:
-        if item.is_junk or item.is_fav:
+        if item.is_junk or (item.is_fav and IniConfigLoader().general.mark_as_favorite):
```

Comment:
Can't we still skip items that are either junk or faved already?

Reply:
True, I can make that change